src/patch_parser.py

```diff
@@ -415,34 +433,41 @@ def _does_match_end_of_super_chunk(lines: InputSource) -> bool:
 def _parse_patch_file_unchanged_chunk(
         lines: InputSource,
         gerrit_orig_line: int,
-        gerrit_new_line: int) -> Tuple[int, int, PatchFileChunkLineMap]:
+        gerrit_new_line: int,
+        parser_state: HunkParserState) -> Tuple[int, int, PatchFileChunkLineMap]:
     in_start = lines.line_number()
     while (not _does_match_end_of_super_chunk(lines)) and (
             (not lines[0]) or (lines[0] and lines[0][0] != '+' and lines[0][0] != '-')):
         logging.info('dropping line: %s', lines[0])
         lines.consume()
         gerrit_orig_line += 1
         gerrit_new_line += 1
+        # logging.warning('Unchanged start: ' + str(in_start))
+        # logging.warning('Unchanged line: ' + lines[0])
+        # logging.warning('Unchanged lines - 1: ' + str(lines.line_number() - 1))
```

Comment:
As I think Daniel mentioned, please drop these comments when you are done or promote them to debug log statements.